### PR TITLE
polish ClientForwardController

### PR DESCRIPTION
**Pull request by mshima**
---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/server/templates/src/main/java/package/web/rest/ClientForwardController.java.ejs [hunk 1]
```diff
@@ -23,22 +23,40 @@ import org.springframework.web.bind.annotation.GetMapping;
 
 @Controller
 public class ClientForwardController {
+    // File names with extension.
+    public static final String EXTENSION_PATTERN = "/{file:[^\\.]*}";
+
+    // Folders with trailing separator.
+    public static final String TRAILING_PATTERN = "/{traillingDir:.*}/";
```
Comment (mraible) [inline generators/server/templates/src/main/java/package/web/rest/ClientForwardController.java.ejs:30]: Is `traillingDir` an expression or can it be changed to `trailingDir`?

File: generators/server/templates/src/main/java/package/web/rest/ClientForwardController.java.ejs [hunk 2]
```diff
@@ -23,22 +23,40 @@ import org.springframework.web.bind.annotation.GetMapping;
...
+<%_ if (communicationSpringWebsocket) { _%>
+
+    // Root folders allow list. Folders that shouls not be forwarded to index.hml.
```
Comment (mraible) [inline generators/server/templates/src/main/java/package/web/rest/ClientForwardController.java.ejs:33]:
```suggestion
    // Root folders allow list. Folders that should not be forwarded to index.html.
```

File: generators/server/templates/src/main/java/package/web/rest/ClientForwardController.java.ejs (hunk 1 above)
Comment (mshima) [reply generators/server/templates/src/main/java/package/web/rest/ClientForwardController.java.ejs:30]: typo
```suggestion
    public static final String TRAILING_PATTERN = "/{trailingDir:.*}/";
```

**Review by mraible — APPROVED** [review]

**Comment by mshima** [comment]
Changed the constants names a bid.
Feel free to change.
Merging to continue inside the sb3 branch.

**Pull request merged**